docs/server/commands.rst

```diff
@@ -901,6 +891,16 @@ run_cherrypy
 Run the CherryPy server bundled with the Translate Toolkit.
 
 
+.. django-admin:: start
+
+start
+^^^^^
+
+.. deprecated:: 2.7.3
```

Comment:
i think this should be "removed" or similar, if its deprecated its still there for now